lib/workload/stateless/schema/component.ts

```diff
@@ -20,23 +11,23 @@ export interface SchemaConstructProps {
 
 export class SchemaConstruct extends Construct {
 
-  constructor(scope: Construct, id: string, cProps: SchemaConstructProps) {
+  constructor(scope: Construct, id: string, props: SchemaConstructProps) {
     super(scope, id);
-    this.createConstruct(cProps);
+    this.createConstruct(props);
   }
 
-  private createConstruct(cProps: SchemaConstructProps) {
+  private createConstruct(props: SchemaConstructProps) {
 
-    const content: string = this.getSchemaContent(cProps.schemaLocation)
+    const content: string = this.getSchemaContent(props.schemaLocation);
 
-    new eventschemas.CfnSchema(this, cProps.schemaName, {
+    new eventschemas.CfnSchema(this, props.schemaName, {
       content: content,
-      registryName: cProps.registryName,
+      registryName: props.registryName,
       type: 'OpenApi3',
```

Comment:
Could push this into props as well. Make it optional and default to this value (possible use case for optional config + defaults)

Reply:
You mean `type: 'OpenApi3'`? Sure, happy to update that right away, yes.

Reply:
Well, since the schema are stateless and defined outside, so this should probably be configurable. 
Or would we want to enforce a particular type?

Reply:
Ok no prob; I will update it to externalise it as well.

Reply:
Added with https://github.com/umccr/orcabus/commit/af853c6097cab635d74df1a23a97bffead6ce212